tests/test_paperqa.py

```diff
@@ -814,6 +814,10 @@ def test_pdf_reader_w_no_match_doc_details(stub_data_dir: Path) -> None:
     )
 
 
+# default: ['method', 'scheme', 'host', 'port', 'path', 'query']
+# this is the default list + body
+# this ensures vcr distinguishes requests with a different body
```

Comment:
I am tripping up on "distinguishes". Do you mean this cassette will:
- Generate a new one if the body changes
- Not generate a new one if the body changes

Reply:
Maybe I can word this better, vcr uses these properties to distinguish how requests are different.
If you look through the cassette, many of the stored requests are identical with the exception of the body (identical method, URI, headers, etc)

for example the calls to https://api.openai.com/v1/embeddings and /v1/chat/completion.

Adding body won't create new cassettes; it just helps vcr correctly match the right request with its response.